cmd/ip.go

```diff
@@ -161,7 +161,7 @@ func ipStatic(args []string) error {
 
 	if !viper.GetBool("yes-i-really-mean-it") {
 		fmt.Println("Turning an IP from ephemeral to static is irreversible. The IP address is not cleaned up automatically on cluster deletion. The address will be accounted until the IP address gets freed manually from your side.")
-		err = helper.Prompt("Are you sure? (y/n)", "y")
+		err = helper.Prompt("Are you sure? (y)", "y")
```

Comment:
Why removing "/n" ?

Reply:
You're right, it should not be part of this PR. I made an issue why I think there is something problematic about the "no" answer: https://github.com/fi-ts/cloudctl/issues/52